KiryakovIA/lesson4/task1.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const consolidate = require('consolidate');
+const path = require('path');
+const request = require('request');
+const iconv = require('iconv-lite');
+const cheerio = require('cheerio');
+
+const app = express();
+
+app.engine('hbs', consolidate.handlebars);
+app.set('view engine', 'hbs');
+app.set('views', path.resolve(__dirname, 'views'));
+
+app.use(express.urlencoded({extended: false}));
+
+
+app.get('/', (req, res) => {
+    res.render('news');
+});
+
+app.post('/settings', (req, res) => {
+    request({ 
+        uri: 'https://www.fontanka.ru/',
+        method: 'GET',
+        encoding: null
+        }, (err, response, body) => {
+        if (!err && response.statusCode == 200) {
+            const strBody = iconv.decode(body, 'win1251');
+            const $ = cheerio.load(strBody);
+            const news = [];
+            
+            $('.sb-item__title').each(function(index) {
+                news.push($(this).text());
+                if (req.body.count && index === req.body.count - 1){
+                    return false;
+                }
+            });
+            res.render('news', { news });
+        }
+        else{
```

Comment:
и перенос строки тоже был лишний
```suggestion
        else {
```